src/main/java/uk/gov/hmcts/probate/service/LifeEventService.java

```diff
@@ -41,6 +45,24 @@ public LifeEventService(final DeathService deathService, final CcdClientApi ccdC
         this.deathRecordService = deathRecordService;
     }
 
+    public DeathRecord getDeathRecordById(final Integer systemNumber) {
+        log.info("Trying LEV call");
+        V1Death v1Death;
```

Comment:
possibly some more relevant name to the var, we are doing operations on this too

Reply:
Renamed variable as "record"